src/App.tsx

```diff
@@ -1,26 +1,103 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { addTodos, deleteTodos, getTodos, USER_ID } from './api/todos';
+import { TodoHeader } from './components/TodoHeader';
+import { TodoFooter } from './components/TodoFooter';
+import { TodoList } from './components/TodoList';
+import { TodoItem } from './components/TodoItem';
+import { Todo } from './types/Todo';
+import { Filters } from './types/Filters';
+import { ErrorNotification } from './components/ErrorNotification';
+import { filterTodos } from './utils/filterTodos';
+import { ErrorType } from './types/ErrorType';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorTodos, setErrorTodos] = useState<ErrorType>(ErrorType.Empty);
+  const [currentFilter, setCurrentFilter] = useState<Filters>(Filters.All);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [loadingTodosIds, setLoadingTodosIds] = useState<number[]>([]);
+
+  const filtered = useMemo(
+    () => filterTodos(todos, currentFilter),
+    [todos, currentFilter],
+  );
+
+  const onAddTodo = async (todoTitle: string) => {
+    setTempTodo({ id: 0, title: todoTitle, completed: false, userId: USER_ID });
```

Comment:
need to put away the previous error message at the beginning of the action

and on the lines 40, 53 too